src/app/commissioner_app.cpp

```diff
@@ -152,7 +152,7 @@ Error CommissionerApp::Start(const std::string &aBorderAgentAddr, uint16_t aBord
 
 void CommissionerApp::Stop()
```

Comment:
let Stop() return error ?

Reply:
I think we can always force stop a Commissioner.